.github/workflows/build.yml

```diff
@@ -186,62 +192,3 @@ jobs:
                 path: /tmp
         -   run: tar -xf /tmp/web3-${{ matrix.node }}.js.tar.gz -C ./
         -   run: yarn build:docs
-
-  coverage-tests-unit:
```

Comment:
These are removed as we can now individually check packages coverage on codecov report page, and by removing these actions CI time will be saved.